ndex: chromium-133.0.6943.53/chrome/browser/ui/lens/lens_overlay_request_id_generator.h
===================================================================
--- chromium-133.0.6943.53.orig/chrome/browser/ui/lens/lens_overlay_request_id_generator.h
+++ chromium-133.0.6943.53/chrome/browser/ui/lens/lens_overlay_request_id_generator.h
@@ -5,6 +5,8 @@
 #ifndef CHROME_BROWSER_UI_LENS_LENS_OVERLAY_REQUEST_ID_GENERATOR_H_
 #define CHROME_BROWSER_UI_LENS_LENS_OVERLAY_REQUEST_ID_GENERATOR_H_
 
+#include <optional>
+
 #include "third_party/lens_server_proto/lens_overlay_routing_info.pb.h"
 #include "third_party/lens_server_proto/lens_overlay_server.pb.h"
 #include "third_party/lens_server_proto/lens_overlay_service_deps.pb.h"
Index: chromium-133.0.6943.53/components/payments/content/payment_method_manifest_table.h
===================================================================
--- chromium-133.0.6943.53.orig/components/payments/content/payment_method_manifest_table.h
+++ chromium-133.0.6943.53/components/payments/content/payment_method_manifest_table.h
@@ -6,6 +6,7 @@
 #define COMPONENTS_PAYMENTS_CONTENT_PAYMENT_METHOD_MANIFEST_TABLE_H_
 
 #include <memory>
+#include <optional>
 #include <string>
 #include <vector>
 
Index: chromium-133.0.6943.53/components/viz/service/display/surface_aggregator.h
===================================================================
--- chromium-133.0.6943.53.orig/components/viz/service/display/surface_aggregator.h
+++ chromium-133.0.6943.53/components/viz/service/display/surface_aggregator.h
@@ -132,13 +132,18 @@ class VIZ_SERVICE_EXPORT SurfaceAggregat
   struct PrewalkResult;
 
   struct AggregateStatistics {
-    int prewalked_surface_count = 0;
-    int copied_surface_count = 0;
+    AggregateStatistics()
+      : prewalked_surface_count(0),
+	copied_surface_count(0),
+	has_pixel_moving_filter(false),
+	has_unembedded_pass(false) {}
+    int prewalked_surface_count;
+    int copied_surface_count;
     // True if the current frame contains a pixel-moving foreground filter
     // render pass.
-    bool has_pixel_moving_filter = false;
+    bool has_pixel_moving_filter;
     // True if the current frame contains a unembedded render pass.
-    bool has_unembedded_pass = false;
+    bool has_unembedded_pass;
 
     base::TimeDelta prewalk_time;
     base::TimeDelta copy_time;
Index: chromium-133.0.6943.53/gpu/command_buffer/service/shared_image/shared_image_backing.h
===================================================================
--- chromium-133.0.6943.53.orig/gpu/command_buffer/service/shared_image/shared_image_backing.h
+++ chromium-133.0.6943.53/gpu/command_buffer/service/shared_image/shared_image_backing.h
@@ -394,7 +394,7 @@ class GPU_GLES2_EXPORT SharedImageBackin
  private:
   class ScopedWriteUMA {
    public:
-    ScopedWriteUMA() = default;
+    ScopedWriteUMA() : content_consumed_(false) {}
 
     ScopedWriteUMA(const ScopedWriteUMA&) = delete;
     ScopedWriteUMA& operator=(const ScopedWriteUMA&) = delete;
@@ -408,7 +408,7 @@ class GPU_GLES2_EXPORT SharedImageBackin
     void SetConsumed() { content_consumed_ = true; }
 
    private:
-    bool content_consumed_ = false;
+    bool content_consumed_;
   };
 
   const Mailbox mailbox_;
Index: chromium-133.0.6943.53/mojo/public/cpp/bindings/lib/multiplex_router.h
===================================================================
--- chromium-133.0.6943.53.orig/mojo/public/cpp/bindings/lib/multiplex_router.h
+++ chromium-133.0.6943.53/mojo/public/cpp/bindings/lib/multiplex_router.h
@@ -315,8 +315,9 @@ class COMPONENT_EXPORT(MOJO_CPP_BINDINGS
   // are not managed by the MultiplexRouter and thus are not relevant here.
   struct ExclusiveSyncWaitInfo {
     InterfaceId interface_id = kInvalidInterfaceId;
-    uint64_t request_id = 0;
-    bool finished = false;
+    uint64_t request_id;
+    bool finished;
+    ExclusiveSyncWaitInfo() : request_id(0), finished(false) {}
   };
   std::optional<ExclusiveSyncWaitInfo> exclusive_sync_wait_;
 
Index: chromium-133.0.6943.53/third_party/blink/renderer/core/layout/inline/line_breaker.h
===================================================================
--- chromium-133.0.6943.53.orig/third_party/blink/renderer/core/layout/inline/line_breaker.h
+++ chromium-133.0.6943.53/third_party/blink/renderer/core/layout/inline/line_breaker.h
@@ -445,9 +445,12 @@ class CORE_EXPORT LineBreaker {
     STACK_ALLOCATED();
 
    public:
-    InlineItemResults* item_results = nullptr;
-    wtf_size_t item_result_index = WTF::kNotFound;
-    const ShapeResultView* collapsed_shape_result = nullptr;
+    TrailingCollapsibleSpace()
+      : item_results(nullptr), item_result_index(WTF::kNotFound),
+	collapsed_shape_result(nullptr) {}
+    InlineItemResults* item_results;
+    wtf_size_t item_result_index;
+    const ShapeResultView* collapsed_shape_result;
     // Ancestors of `item_result`. ancestor_ruby_columns[0] is the parent of
     // `item_result`, and ancestor_ruby_columns[n+1] is the parent of
     // ancestor_ruby_columns[n]. This list is empty if `item_result` is not
Index: chromium-133.0.6943.53/third_party/blink/renderer/core/paint/paint_layer_scrollable_area.h
===================================================================
--- chromium-133.0.6943.53.orig/third_party/blink/renderer/core/paint/paint_layer_scrollable_area.h
+++ chromium-133.0.6943.53/third_party/blink/renderer/core/paint/paint_layer_scrollable_area.h
@@ -888,8 +888,9 @@ class CORE_EXPORT PaintLayerScrollableAr
 
   struct PendingViewState {
     HistoryItem::ViewState state;
-    mojom::blink::ScrollBehavior scroll_behavior =
-        mojom::blink::ScrollBehavior::kAuto;
+    mojom::blink::ScrollBehavior scroll_behavior;
+    PendingViewState()
+      : scroll_behavior(mojom::blink::ScrollBehavior::kAuto) {}
   };
   std::optional<PendingViewState> pending_view_state_;
 };
Index: chromium-133.0.6943.53/third_party/blink/renderer/modules/media_controls/elements/media_control_timeline_element.h
===================================================================
--- chromium-133.0.6943.53.orig/third_party/blink/renderer/modules/media_controls/elements/media_control_timeline_element.h
+++ chromium-133.0.6943.53/third_party/blink/renderer/modules/media_controls/elements/media_control_timeline_element.h
@@ -51,7 +51,8 @@ class MediaControlTimelineElement : publ
   // Struct used to track the current live time.
   struct LiveAnchorTime {
     base::TimeTicks clock_time_;
-    double media_time_ = 0;
+    double media_time_;
+    LiveAnchorTime() : media_time_(0) {}
   };
 
   void DefaultEventHandler(Event&) override;
Index: chromium-133.0.6943.53/third_party/webrtc/call/rtp_config.h
===================================================================
--- chromium-133.0.6943.53.orig/third_party/webrtc/call/rtp_config.h
+++ chromium-133.0.6943.53/third_party/webrtc/call/rtp_config.h
@@ -79,10 +79,12 @@ struct RtpStreamConfig {
   struct Rtx {
     std::string ToString() const;
     // SSRC to use for the RTX stream.
-    uint32_t ssrc = 0;
+    uint32_t ssrc;
 
     // Payload type to use for the RTX stream.
-    int payload_type = -1;
+    int payload_type;
+
+    Rtx() : ssrc(0), payload_type(-1) {}
   };
   std::optional<Rtx> rtx;
 };


